src/scripts/app.ts

```diff
@@ -334,11 +333,13 @@ export class ComfyApp {
         if (ComfyApp.clipspace.images) {
           const clip_image =
             ComfyApp.clipspace.images[ComfyApp.clipspace['selectedIndex']]
-          const index = node.widgets.findIndex((obj) => obj.name === 'image')
+          const index = node.widgets.findIndex(
+            (widget: IWidget) => widget.name === 'image'
+          )
           if (index >= 0) {
             if (
-              node.widgets[index].type != 'image' &&
-              typeof node.widgets[index].value == 'string' &&
+              node.widgets[index].type === 'string' &&
```

Comment:
Why `!= 'image'` condition is changed here?

I don't think we need to maintain clip space related code as the feature is planned to be removed. We shall have an RFC published discussing the removal/replacement of the feature in https://github.com/Comfy-Org/rfcs. Stay tuned there.